prime-router/docs/vpn.md

```diff
@@ -41,6 +41,39 @@ $ kill -${SIGNAL} $(cat "/tmp/${USER}/openvpn.pid")
 # On termination, openvpn will remove the PID file
 ```
 
+## Windows
+### VPN DNS Resolution
+There is an issue where the DNS server for the VPN adapter is not used when resolving hostnames.  This is due to the interface metric of the VPN loosing to the metric of your normal network adapter.  Run the following command to look up the IP of a server in the Azure environment and test if this is an issue .  For example:
+
+`nslookup pdhstaging-pgsql.postgres.database.azure.com`
+
+should return a 10.x.x.x IP address which is in the range used by the VPN.  If you see an address outside of this 10.x.x.x range then continue with the instructions here to fix this issue.
```

Comment:
Consider using CIDR notation (i.e. `10.0.0.0/8` instead of `10.x.x.x`)
```suggestion
should return a 10.0.0.0/8 IP address which is in the range used by the VPN.  If you see an address outside of this 10.0.0.0/8 range then continue with the instructions here to fix this issue.
```

Reply:
> Consider using CIDR notation (i.e. `10.0.0.0/8` instead of `10.x.x.x`)

I would like to use something that people that do not understand that notation can follow.  Something simple.  Any other ideas?

Reply:
I think it's fine to require folks to understand CIDR notation, especially given that this is a topic about networking.
If we were talking about `/20` or other uncommon masks then sure, but `/8`, `/16`, `/24` are pretty common and easily grokable.

Reply:
Makes sense.  Will change.